channel-email/src/main/java/com/synopsys/integration/alert/channel/email/web/EmailGlobalConfigController.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * channel-email
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.channel.email.web;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.rest.ResponseFactory;
+import com.synopsys.integration.alert.common.rest.api.BaseResourceController;
+import com.synopsys.integration.alert.common.rest.api.TestController;
+import com.synopsys.integration.alert.common.rest.api.ValidateController;
+import com.synopsys.integration.alert.common.rest.model.ValidationResponseModel;
+
+@RestController
+@RequestMapping(AlertRestConstants.EMAIL_CONFIGURATION_PATH)
+public class EmailGlobalConfigController implements BaseResourceController<EmailGlobalConfigResponse>, TestController<EmailGlobalConfigResponse>, ValidateController<EmailGlobalConfigResponse> {
```

Comment:
I wonder if we could also create a `ChannelGlobalConfigController` superclass that implements the relevant interfaces and handles all of the calls to `ResponseFactory.createContentResponseFromAction(...)`. Not for this PR, but I wonder if there is ever a case where we don't do that.